File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    let answer = []
    let arr = s.split(' ')
    for (let i = 0; i < arr.length; i++) {
        let word = ''
        for (let j = 0; j < arr[i].length; j++) {
            if (j % 2 === 0) {
                word += arr[i][j].toUpperCase()
            } else {
                word += arr[i][j].toLowerCase()
            }
        }
        answer.push(word)
    }
    return answer.join(' ')
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer_words = []
    words = s.split(' ')
    for word in words:
        transformed_word_chars = []
        for j, char in enumerate(word):
            if j % 2 == 0:
                transformed_word_chars.append(char.upper())
            else:
                transformed_word_chars.append(char.lower())
        answer_words.append("".join(transformed_word_chars))
    return " ".join(answer_words)